test/integration/revocation_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/eggsampler/acme/v3"
```

Comment:
The existing TestPrecertificateRevocation test was able to use the integration test helpers we have: makeClient, authAndIssue, etc. Could you briefly summarize here what it was about this test that needed to use the eggsampler/acme library directly?